syntax = "proto3";

// Model 消息,框架内部消息
package Model;


// 消息头
message MessageHead	// IResponse
{
	int32 RpcId = 90;
	int32 Error = 91;
	string Message = 92;
	int64 ActorId = 93;
}



message TestRequest
{
	int32 A = 1;
	int32 B = 2;
	
	int32 RpcId = 90;
}

message TestResponse
{
	int32 Sum = 1;
	
	int32 RpcId = 90;
	int32 Error = 91;
	string Message = 92;
}


message ActorTestRequest
{
	int32 A = 1;
	int32 B = 2;
	
	int32 RpcId = 90;
	int64 ActorId = 93;
}

message ActorTestResponse
{
	int32 Sum = 1;
	
	int32 RpcId = 90;
	int32 Error = 91;
	string Message = 92;
}